<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateStaff</class>
 <widget class="QDialog" name="UpdateStaff">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>631</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Update Staff</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>611</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Current Details</string>
    </property>
    <widget class="QLabel" name="currentNameLabel">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>40</y>
       <width>601</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Name : </string>
     </property>
    </widget>
    <widget class="QLabel" name="currentEmailLabel">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>60</y>
       <width>601</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Email :</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentContactNumberLabel">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>80</y>
       <width>601</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Contact Number :</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentAddressLabel">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>100</y>
       <width>601</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Address :</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentShopLabel">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>120</y>
       <width>601</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Shop :</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentAdminRightsLabel">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>140</y>
       <width>601</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Admin Rights : </string>
     </property>
    </widget>
    <widget class="QLabel" name="currentIDLabel">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>20</y>
       <width>601</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ID :</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>611</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Update Personal Details</string>
    </property>
    <widget class="QLineEdit" name="updatePersonalDetailsLineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>137</width>
       <height>36</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="personalDetailsComboBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>171</width>
       <height>36</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Name</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Email</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Contact Number</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Address</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="updatePersonalDetailsButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>88</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>611</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Update Admin</string>
    </property>
    <widget class="QComboBox" name="updateAdminComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>77</width>
       <height>36</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Yes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="updateAdminButton">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>20</y>
       <width>88</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>88</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>88</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>180</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="emailLineEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
